src/integration.ts

```diff
@@ -20,7 +20,7 @@ export const isIntegration = <I extends IntegrationInput<string, AnyFunction>>(
   i: any
 ): i is I => typeof i === "object" && "kind" in i;
 
-export type IntegrationCallExpr = Omit<CallExpr, "expr"> & {
+export type IntegrationCallExpr = CallExpr & {
   expr: ReferenceExpr<Integration>;
 };
```

Comment:
If you don't omit, it will union the expr as `undefined | ReferenceExpr` or at least I thought.

Reply:
It won't union, it will intersect, so `expr` can only be a `ReferenceExpr<Integration>`